proto/ibc/applications/middleware/fee/v1/fee.proto

```diff
@@ -0,0 +1,12 @@
+syntax = "proto3";
+
+package ibc.applications.middleware.fee.v1;
+import "cosmos/base/v1beta1/coin.proto";
+option go_package = "github.com/cosmos/ibc-go/modules/apps/29-fee/types";
+
+// Fee interface
+// See Fee Payment Middleware spec:
+// https://github.com/cosmos/ibc/tree/master/spec/app/ics-029-fee-payment#fee-middleware-contract
+message Fee {
+  cosmos.base.v1beta1.Coin amount = 1;
```

Comment:
We can have this be `Coins`, there's an example in the sdk proto files of how to do this. I think you need to use gogo proto to cast it to `Coins`

Reply:
I'm doing it the same way with the interchain acc helper module. However, I get a bunch of linter errors when I try it here. Any ideas?

```
repeated cosmos.base.v1beta1.Coin amount = 3
      [(gogoproto.nullable) = false, (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins"];
```

` unknown extension gogoproto.nullable`
` unknown extension gogoproto.castrepeated`
`unknown extension gogoproto.moretags`

Reply:
Add `import "gogoproto/gogo.proto";` to the top of the file